### PR TITLE
Update contents

### LINKED ISSUE #60: Review docs
### Description:

The following docs has to be reviewed on formatting, outdated and incorrect or invalid contents.
https://mg.docs.wso2.com/en/latest/getting-started/overview/ 
https://mg.docs.wso2.com/en/concepts/developer-first-apis/ 
https://mg.docs.wso2.com/en/concepts/openapi-extensions/ 
https://mg.docs.wso2.com/en/how-tos/add-a-grpc-api/#microgateway-extensions
https://mg.docs.wso2.com/en/how-tos/security/api-authentication/mutual-ssl-authentication/
https://mg.docs.wso2.com/en/how-tos/rate-limiting/rate-limiting-overview/
https://mg.docs.wso2.com/en//reference/default-ports/ 

### Content Positioning in Documentation:

- Link:

- Heading (& Step):

- Any other reference:

---

### PR DESCRIPTION
**Pull request by AmaliMatharaarachchi**
### Purpose

Fixes issues in #60
Update and fix contents on mg docs.
Introduce {{apim_path}} to urls

Remove duplicated content "Adding Debug Logs" from troubleshooting doc [1][2].
[1] https://mg.docs.wso2.com/en/latest/troubleshooting/troubleshooting/#adding-debug-logs
[2] https://mg.docs.wso2.com/en/latest/troubleshooting/adding-debug-logs/

### Issues

Fixes #60

---
#### Maintainers: Check before merge
- [ ] Manual verification

### CODE REVIEW

File: en/docs/how-tos/security/api-authorization/scope-validation.md [hunk 1]
```diff
@@ -71,9 +71,9 @@ paths:
     get:
       security:
         - mybasic: []
-        - OAuth2:
-        - read
-        - write
+            - OAuth2:
```
[the remark this answers is not in the document]
Comment (praminda) [reply en/docs/how-tos/security/api-authorization/scope-validation.md:74]: Dont need to change everything. Just keep one indentation in a sample or file. seems this sample has both 2, 4 space indentations

File: en/docs/how-tos/configure-caching.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ cacheTimeoutInSeconds: 300
 
 ### Configuring OAuth Cache
 
-The OAuth token is enabled by default in the Microgateway. Adding the following section to the micro-gw.conf file which is located in the `         <MGW-RUNTIME-HOME>/conf        ` directory , you can change the configurations in the OAuth cache.
+The OAuth token is enabled by default in the Microgateway. Adding the following section to the `micro-gw.conf` file which is located in the `         <MGW-RUNTIME-HOME>/conf        ` directory , you can change the configurations in the OAuth cache.
```
Comment (praminda) [inline en/docs/how-tos/configure-caching.md:24]:
```suggestion
The OAuth token is enabled by default in the Microgateway. Adding the following section to the `micro-gw.conf` file which is located in the `<MGW-RUNTIME-HOME>/conf` directory , you can change the configurations in the OAuth cache.
```

File: en/docs/concepts/openapi-extensions.md [hunk 3]
```diff
@@ -5,18 +5,18 @@ WSO2 API Microgateway supports the following OpenAPI Extensions. You can use the
...
 | `                           x-wso2-cors                         `                     | Specifies the Cross-Origin Resource Sharing (CORS) configuration for the API.                                          | Optional → API level only      |
-| `             x-wso2-endpoints            `                                           | Defines the endpoint configuration globally which thereafter can be referred to using one of the following extensions.| `x-wso2-production-endpoints`   ` x-wso2-sandbox-endpoints`                                              | Optional                       |
-| `                           x-wso2-disable-security                         `         | Enables the resource to be invoked without authentication.                                                             | Optional → Resource level only |
+| `             x-wso2-endpoints            `                                           | Defines the endpoint configuration globally which thereafter can be referred to using one of the following extensions. <li>`x-wso2-production-endpoints`</li> <li>` x-wso2-sandbox-endpoints`</li> | Optional |
```
Comment (praminda) [inline en/docs/concepts/openapi-extensions.md:14]: Shall we fix this in all places in the table?

```suggestion
| `x-wso2-endpoints`                                           | Defines the endpoint configuration globally which thereafter can be referred to using one of the following extensions. <li>`x-wso2-production-endpoints`</li> <li>` x-wso2-sandbox-endpoints`</li> | Optional |
```

File: en/docs/troubleshooting/troubleshooting.md [hunk 4]
```diff
@@ -231,45 +220,3 @@ Given below are some WSO2 API Microgateway specific error codes and their meanin
 | `                  900804                 ` | Subscription level throttled out.                                                                                                                                                                                  |
 | `                  900808                 ` | An internal error occurred in the Microgateway.                                                                                                                                                                    |
 | `                  900809                 ` | An internal error occurred in the Microgateway, since a subscription or application throttle policy is not deployed. This might be due to adding a throttle policy to API-M and not regenerating the Microgateway. |
-
```
[the remark this answers is not in the document]
Comment (AmaliMatharaarachchi) [reply en/docs/troubleshooting/troubleshooting.md:234]: It was duplicated in [2].
[1] https://mg.docs.wso2.com/en/latest/troubleshooting/troubleshooting/#adding-debug-logs
[2] https://mg.docs.wso2.com/en/latest/troubleshooting/adding-debug-logs/

File: en/docs/how-tos/security/api-authorization/scope-validation.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (AmaliMatharaarachchi) [reply en/docs/how-tos/security/api-authorization/scope-validation.md]: Fixed https://github.com/wso2/docs-mg/pull/63/commits/a66ea0eb0929d5235d21bac0085ffb02a04e9bda

**Review by shaniR — APPROVED** [review]

**Pull request merged**